### PR TITLE
Support you need during the case part-2

### PR DESCRIPTION
**Pull request by swaroopa86**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-1538

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/applicant/support-you-need-during-case/travelling-to-court/template.njk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (cshiwarale) [inline src/main/steps/applicant/support-you-need-during-case/travelling-to-court/template.njk:1]: Don't we need section and title to be shown for this page?

File: src/main/steps/applicant/support-you-need-during-case/communication-help/content.ts [hunk 2]
```diff
@@ -0,0 +1,175 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { atLeastOneFieldIsChecked, isFieldFilledIn, isTextAreaValid } from '../../../../app/form/validation';
+
+const en = {
+  section: 'Reasonable adjustments',
+  title: 'I need help communicating and understanding',
+  courtcommunication:
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/steps/applicant/support-you-need-during-case/communication-help/content.ts:8]: comment addressed

File: src/main/modules/properties-volume/index.ts [hunk 3]
```diff
@@ -12,30 +12,40 @@ export class PropertiesVolume {
...
   }
-
   /**
    * Load a secret from the AAT vault using azure cli
    */
-  private setLocalSecret(secret: string, toPath: string): void {
-    const result = execSync(`az keyvault secret show --vault-name prl-aat -o tsv --query value --name ${secret}`);
-    set(config, toPath, result.toString().replace('\n', ''));
+
+  pri; //vate setLocalSecret(secret: string, toPath: string): void {
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/modules/properties-volume/index.ts:36]: comment addressed

File: src/main/app/form/Form.ts [hunk 4]
```diff
@@ -182,12 +182,14 @@ export interface FormInput {
   warning?: Warning;
   conditionalText?: Label;
   subFields?: Record<string, FormField>;
-  divider?: Label; //Required for divider between checkbox options
+  //divider?: Label; //Required for divider between checkbox options
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/app/form/Form.ts:185]: comment addressed

File: src/main/app/case/case.ts [hunk 5]
```diff
@@ -89,6 +89,23 @@ export interface Case {
...
+  languageRequirements?: string;
+  languageDetails?: string;
+  reasonableAdjustments?: string;
+  helpcommunication?: string;
+  describeOtherNeed?: string;
+  courthearing?: string;
+  communicationSupportOther?: string;
+  docsSupport?: string;
+  otherDetails?: string;
+  courtcomfort?: string;
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/app/case/case.ts:101]: comment addressed

File: src/main/app/case/case.ts [hunk 6]
```diff
@@ -89,6 +89,23 @@ export interface Case {
...
+  otherProvideDetails?: string;
+  safetyArrangements?: string;
+  safetyArrangementsDetails?: string;
+  travellingtoCourt?: string;
+  travellingOtherDetails?: string;
+  unableforcourtproceedings?: string;
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/app/case/case.ts:107]: comment addressed

File: src/main/app/case/case.ts [hunk 7]
```diff
@@ -89,6 +89,23 @@ export interface Case {
   applicant1AddressPostcode?: string;
   applicant1ContactDetails?: ContactDetails[];
   applicant1ContactDetailsConsent?: YesOrNo;
+  languageRequirements?: string;
+  languageDetails?: string;
+  reasonableAdjustments?: string;
+  helpcommunication?: string;
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/app/case/case.ts:95]: comment addressed

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**